src/deploy/GoogleDeployer.js

```diff
@@ -38,7 +38,7 @@ export default class GoogleDeployer extends BaseDeployer {
         return false;
       }
     }
-    return !!this._client && !!this._cfg.projectID && !!this._cfg.keyFile;
+    return !!this._cfg.projectID && !!this._cfg.keyFile;
```

Comment:
check for required configuration, client is only created in `init` (see other deployers)